docs/userGuide/usingPlugins.md

```diff
@@ -1,4 +1,8 @@
+<variable name="title">Using Plugins</variable>
+<variable name="filename">usingPlugins</variable>

```

Comment:
If the `{{ filename }}` variable isn't used, we should not declare it. 

See `tipsAndTricks.md`

Reply:
Fixed!